tests/test_class_sh_property.cpp

```diff
@@ -0,0 +1,90 @@
+#include "pybind11_tests.h"
+
+#include "pybind11/smart_holder.h"
```

Comment:
I've seen
`#include <pybind11/smart_holder.h>`
Elsewhere is there a consistent choice we should make with these includes?